client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -96,12 +101,12 @@ const Checkout = React.createClass( {
 	},
 
 	addProductToCart: function() {
-		var planSlug = this.props.plans.getSlugFromPath( this.props.product ),
+		var planSlug = getUpgradePlanSlugFromPath( this.props.product ),
```

Comment:
I want to keep changes to minimum and I desperately want to avoid sideffects in the checkout flow.